### ISSUE TITLE
Set up Zcash AWS account.

### ISSUE THREAD

**Issue by defuse** [zcash/zcash#262]
I have the company credit card info, so we should create the account in a pairing session.

**Comment by mlbaily** [comment]
Don't forget to add my number.

**Comment by daira** [comment]
I think @defuse got unassigned from this due to the org change.

**Comment by defuse** [comment]
Yes, I was unassigned due to the org change. Yes, I still have to add Margaret's number.

**Comment by str4d** [comment]
Woawoaaooaa. I

**Comment by defuse** [comment]
@mlbaily I'll need to coordinate a time to do this with you, it'll probably want to text/call you a code and have me type it in.

**Comment by defuse** [comment]
Hm, @mlbaily's phone number is already on the account, and it's the only one visible in any of the settings. I think maybe Amazon just wanted to verify that *a* phone number existed when I created the account. It's hard to know what Amazon's policy would be if in a year I came to them saying "I lost my password, but I can show you 10 pieces of government photo ID and if you check your logs you'll see my phone number as the one that created the account." I wish these policies were always explicit and easy to find. Searching for "amazon aws password reset" and "amazon aws account change password" and "amazon aws forgot password" turn up nothing useful.
